about.html

```diff
@@ -0,0 +1,89 @@
+      <a class="nav-item" href="#">Program</a>
+      <a class="nav-item" href="#">Join</a>
+      <a class="nav-item" href="#">Sponser</a>
+      <a class="nav-item" href="#">News</a>
+      <a class="nav-item" href="#">AI Compaign</a>
+    </div>
+  </header>
+<section class="about">
+  <h1 class="aboutTitle">MACHIN LEARNING<br>ONLONE COURSES 2022</h1>
+  <p class="abou-textBox">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed molestie ante, a semper magna. Nam quis iaculis libero. Sed vehicula, augue et dapibus ornare, magna risus aliquet ipsum, in imperdiet ex arcu ac risus. Cras porta aliquam porttitor. Nunc sed tortor ipsum. Cras suscipit ex risus, sed consequat quam aliquet id. Cras hendrerit suscipit odio, sit amet venenatis eros vestibulum consectetur. Integer at enim sagittis, lacinia massa nec, condimentum ex.</p>
+  <p class="about-normal-text">Please contact us per Email for any further question about machin learnng online coures !</p>
+  <a href="#">machinlearning2022@gmail.com <img src="https://img.icons8.com/external-those-icons-lineal-those-icons/10/000000/external-right-arrows-those-icons-lineal-those-icons-1.png" alt="close"/></a>
+</section>
+<section class="logo-section">
+  <h2 class="lecturerTitle">Machin Learning Online Course 2022 Logo </h2>
+  <div class="red-line"></div>
+  <p class="about-normal-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent auctor dui eget purus interdum egestas. Aliquam quis dolor pharetra, blandit turpis non, rhoncus mauris. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut lobortis arcu vel porttitor maximus. Vestibulum non tellus vel ligula finibus posuere a a felis.</p>
+  <div class="logo-border"><img id="ml-logo" src="images/logo-large.png" alt="courseLogo"></div>
+</section>
+<section class="gallery">
+  <h2 class="lecturerTitle">See The past courses </h2>
+  <div class="red-line"></div>
+  <p class="about-normal-text" id="gallery-caption">Take a look at the last two machin learning workshop</p>
+  <div class="photoes">
+    <div class="photo-fram">
+    <img src="images/workshop1.jpg" class="photo" alt="workshop2021">
+      <div class="layer">
+        <h1 class="photo-caption">Ml workshop 2021</h1>
+      </div>
+    </div>
+    <div class="photo-fram">
+    <img src="images/workshop2.jpg" class="photo" alt="workshop2020">
+      <div class="layer">
+        <h1 class="photo-caption">Ml workshop 2022</h1>
+      </div>
+      </div>
+  </div>
+</section>
+<section class="license" id="darkLicense">
+  <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">
+    <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" />
+  </a>
+  <br />
+  This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.
```

Comment:
- [ ] The bottom part of the about page in the mobile view has the `partners` section and the `footer` section but yours only has the `footer` section. It would be great if you could arrange that.
- Your design:
![C8](https://user-images.githubusercontent.com/85113567/153273222-7eee75ce-5862-4203-b244-5e5183abe484.PNG)
- Required design:
![C6](https://user-images.githubusercontent.com/85113567/153273274-dd914566-ea52-48ef-9bc1-54e8d81dae83.PNG)